ios/Handlers/PinHandler.swift

```diff
@@ -201,15 +174,12 @@ extension PinHandler: ONGChangePinDelegate {
     }
     
     func userClient(_ userClient: ONGUserClient, didStartPinChangeForUser userProfile: ONGUserProfile) {
-        handleFlowUpdate(PinFlow.change, error: nil, receiver: self, profileId: userProfile.profileId, userInfo: nil, data: nil)
+        // handle change pin start if needed
```

Comment:
Does anything need to happen here? Not entirely sure based on this comment.

Reply:
I changed it to a TODO because this is part of a larger refactor that needs to happen to this code.